lib/components/viewers/pattern-row.tsx

```diff
@@ -73,31 +73,37 @@ class PatternRow extends Component<Props, State> {
 
     const routeName = route.shortName ? route.shortName : route.longName
     const routeColor = getRouteColorBasedOnSettings(route.operator, route)
+    console.log(stopTimes)
+    let style
+    if (routeName && routeName?.length >= 4 && routeName?.length <= 6) {
+      style = { fontSize: '20px' }
```

Comment:
Instead of setting the entire style object, we should instead set a font size variable we insert later